cfme/tests/configure/test_docs.py

```diff
@@ -84,8 +83,21 @@ def docs_info():
         return []
 
 
+@pytest.mark.tier(3)
```

Comment:
Fixtures shouldn't be marked with the `tier` marker, only tests, so please remove the tier marker from here and from the `docs_info` fixture as well.
Also, I assume you were inspired by the `docs_info` fixture above when creating `docs_content` but static data usually should be simply stored as constant data in the module, not inside a session-scoped fixture. The difference with `docs_info` is that there is version-dependent data in there and we don't want to run version.pick on module import because it needs to connect to a running appliance to decide the version, so it needs to be in a fixture which is not run on module import. Your static data inside `docs_content` could be stored just as a constant dict, but... since we can safely assume that it will need to be version picked in the near future (with next release) lets keep it as a session-scoped fixture. :+1: So just remove the unneeded markers, please, and this can be merged.